lib/populate/extract-csv.js

```diff
@@ -39,8 +40,8 @@ function extractData(rows, codeCommune) {
       return {
         _id: new ObjectId(),
         commune: codeCommune,
-        nom: toponymeRows[0].parsedValues.voie_nom,
-        nomAlt: toponymeRows[0].localizedValues.voie_nom || null,
+        nom: beautifyUppercased(toponymeRows[0].parsedValues.voie_nom),
+        nomAlt: beautifyNomAlt(toponymeRows[0].localizedValues.voie_nom),
```

Comment:
Attention en l'état `beautifyNomAlt` renvoi `undefined` et non `null`.
```suggestion
        nomAlt: toponymeRows[0].localizedValues.voie_nom ? beautifyNomAlt(toponymeRows[0].localizedValues.voie_nom) : null,
```